### ISSUE TITLE
Support ES6-style import commands on the client side

### ISSUE THREAD

**Issue by iangilman** [opentypejs/opentype.js#416]
My project uses Babel and Webpack, so all of my client side code uses `import` to load dependencies. 

I did an `npm install opentype.js`, and added this to one of my files:

```
import opentype from 'opentype.js`;
console.log(opentype);
```

...and I get `undefined` in the console. If I try to call `opentype.load` I get a "load is not a member of undefined" error.

Using `import` like this works for most other libraries. Am I doing something wrong here, or does opentype just not support this scenario?

Thank you for your consideration, and for an awesome library!

**Comment by rdazvd** [comment]
Looking at the source of the exported opentype.js, I have found no default nor named exports on which Webpack could rely on to understand these values. You are better off inserting a `script` tag inside the `head` of your `index.html` and pointing its source attribute to one of the files inside the `dist` sub folder inside the opentype.js node module ─ by doing so, you don't have to worry about importing opentype.js inside your javascript components as long as the Webpack bundle is loaded somewhere within the `body`.

**Comment by iangilman** [comment]
Oh, good point! It hadn't occurred to me to try importing just the function I needed. I've given that a try now:

```
import { load } from 'opentype.js';
console.log(load);
```

...and it works! So, this solves my issue (though your suggestion to include it as a script tag would also work).

It might still be nice to export a default if possible, so people can import the whole thing if they want.

Anyway, thank you for pointing me in the right direction!

**Comment by miracle2k** [comment]
@Jolg42 This change breaks my app which relies on the existing export structure. It probably should be released as a major version upgrade, if you are doing semver.

Beyond that, and I am not an expert on this subject, though I know it has caused many issues in the past, it may be worth considering to offer both, as they do in react for example.

This may also involve questions of tree shaking, i.e. whether my bundler can ignore those parts of opentype I am not using. This will certainly not work with a default import. Note that it was previously of course possible to do:

```
import * as opentype from 'opentype.js'
```

to have the same result as the default export.

**Comment by Jolg42** [comment]
@miracle2k Thanks for the feedback I didn't know it would break in this case 🤔 

Or maybe I just forgot something I guess, I'll check that!

**Comment by Jolg42** [comment]
@miracle2k That should be fixed in latest version 1.3.2 😉

**Comment by wtagain** [comment]
> @miracle2k That should be fixed in latest version 1.3.2 😉

Hi, when can we get the new version? 😉
